include/ppx/grfx/vk/vk_command.h

```diff
@@ -70,13 +70,25 @@ class CommandBuffer
         uint32_t                          setCount,
         const grfx::DescriptorSet* const* ppSets) override;
 
+    virtual void SetGraphicsPushConstants(
+        const grfx::PipelineInterface* pInterface,
+        uint32_t                       count,
+        const void*                    pValues,
```

Comment:
So, this means that the caller passes a contiguous memory region of 32-bit words, right?  Alignment and padding cannot be checked here, but perhaps this can be documented?

Reply:
For BW shaders, we compile with `-fvk-use-dx-layout` on. Which means allow tight packing on everything except for arrays. There may be edge cases.

I've added note to that function in `graphics_command.h` about Vulkan packing rules.